index.html

```diff
@@ -150,65 +152,134 @@ <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="lang">
+                  <img src="images/icon-frameworks.png" alt="Frameworks" />
+                  <h2>Frameworks</h2>
+                  <div class="languagedefcontain">
+                    <a class="languagesdef2" href="#">Bootstrap</a>
+                    <a class="languagesdef2" href="#">Ruby on rails</a>
+                    <a class="languagesdef2" href="#">RSpec</a>
+                    <a class="languagesdef2" href="#">Capybara</a>
+                    <a class="languagesdef2" href="#">Selenium</a>
                   </div>
                 </div>
-                <div class="card">
-                  <div class="lang">
-                    <img src="images/icon-skills.png" alt="Languages" />
-                    <h2>Skills</h2>
-                    <div class="languagedefcontain">
-                      <a class="languagesdef2" href="#">Codekit</a>
-                      <a class="languagesdef2" href="#">GitHub</a>
-                      <a class="languagesdef2" href="#">Codepen</a>
-                      <a class="languagesdef2" href="#">Gitlab</a>
-                      <a class="languagesdef2" href="#">Terminal</a>
-                    </div>
+              </div>
+              <div class="card">
+                <div class="lang">
+                  <img src="images/icon-skills.png" alt="Languages" />
+                  <h2>Skills</h2>
+                  <div class="languagedefcontain">
+                    <a class="languagesdef2" href="#">Codekit</a>
+                    <a class="languagesdef2" href="#">GitHub</a>
+                    <a class="languagesdef2" href="#">Codepen</a>
+                    <a class="languagesdef2" href="#">Gitlab</a>
+                    <a class="languagesdef2" href="#">Terminal</a>
                   </div>
                 </div>
               </div>
             </div>
           </div>
-        </section>
+        </div>
+      </section>
+      <section class="contact-me">
+        <img
+          class="contact-shape"
+          src="images/illustration-contact-form.png"
+          alt="contact"
+        />
+        <h2 class="contact-text">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form
+          class="contact-form"
+          action="https://formspree.io/f/mlezeeov"
+          method="post"
+        >
+          <input
+            class="Form-Name"
+            name="name"
+            type="text"
+            maxlength="30"
+            placeholder="Full Name"
+            required
+          />
+          <input
+            class="Form-Email"
+            name="Email"
+            type="email"
+            placeholder="Email Address"
+            required
+          />
+          <textarea
+            class="Form-text"
+            name="text"
+            maxlength="500"
+            placeholder="Enter Text Here"
+            required
```

Comment:
- [ ] One of the requirements says that you should stick to the design templates in Figma as much as possible, this includes the font, colors, images, and spacing (margin/padding). Kindly let's go ahead a  make our form look exactly like the template one. Refer yourself to these screenshots below  :point_down: :point_down: 

 Your design | The template
-----------------|------------------
![Screenshot from 2021-12-07 18-44-10](https://user-images.githubusercontent.com/65068771/145071385-21476aad-2904-404b-92c9-993d4f4484f3.png) | ![Screenshot from 2021-12-07 18-40-54](https://user-images.githubusercontent.com/65068771/145071332-eb2b6e01-0ac6-43b5-bd5f-82d7cf95347c.png)